otterdog/eclipse-tractusx.jsonnet

```
@@ -404,6 +404,28 @@ orgs.newOrg('automotive.tractusx', 'eclipse-tractusx') {
         },
       ],
     },
+    orgs.newRepo('engineering-use-case-demonstrator') {
```

```suggestion
    orgs.newRepo('engineering-use-case-demonstrator') {
```
does the name needs to be like this? 

can it be called, engineering-3d-hub or something like this

---

@bosserf, @FEILSTE it's "your" repository :) what do you think?